### DISCUSSION TITLE
Code Pilot not working anymore with a student account

### DISCUSSION THREAD

**Discussion by VladDumitru06** [community/community#43661] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I think it's a bug, i've searched around and saw that relogin into vscode and restarting the PC should fix it but it didn't work. I have a registered student account but when I go to the coPilot page I am asked to pay even though I should be redirected since I have a student account. In my VS code I also get this:
[INFO] [auth] [2023-01-09T00:56:14.733Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-01-09T00:56:14.737Z] GitHub Copilot could not connect to server. Extension activation failed: "No access to GitHub Copilot found."

Which gave me the impression of some token that might have expired but again after relogging the same message is shown again.

**Comment by jongwooo** [comment, marked as answer]
@VladDumitru06 [Others](https://github.com/community/community/discussions/43650) are also raising similar issues. It seems to be a problem with GitHub. I hope it is resolved quickly.

**Reply by anzal1** [reply]
Same issues are being faced by many of my colleagues and on my student account too.

**Comment by otkrickey** [comment]
Solved: #43634

**Reply by dwijbavisi** [reply]
Not solved. Still not working for many, including me.

**Reply by Azzani3** [reply]
Indeed, I still have the issue as well

**Reply by matt-gribben** [reply]
I also still have this issue

**Comment by Yusufihsangorgel** [comment]
Here is my problem currently : "Thank you for using GitHub Copilot. Your subscription has ended." , but Im verified student.

**Reply by Yusufihsangorgel** [reply]
[INFO] [auth] [2023-01-09T08:10:58.876Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-01-09T08:10:58.879Z] GitHub Copilot could not connect to server. Extension activation failed: "Thank you for using GitHub Copilot. Your subscription has ended."

**Reply by matt-gribben** [reply]
Same

**Reply by diegomoralesjimenez** [reply]
Same!

**Reply by FakherKrc** [reply]
same im student

**Reply by Ekbal41** [reply]
same here 😔

**Reply by ky-nguyen** [reply]
ahhh, I have the same, So I'm logging into my GitHub account to sign in and requesting a free GitHub copilot.

**Comment by outsider987** [comment]
same here
